main/urls.py

```diff
@@ -125,6 +127,8 @@
     url(r'^api/v2/add_subscription/', AddSubscription.as_view()),
     url(r'^api/v2/del_subscription/', DelSubscription.as_view()),
     url(r'^api/v2/add_cronjob_log/', AddCronJobLog.as_view()),
+    url(r'^api/v2/event/(?P<pk>\d+)', api_views.EventViewset.as_view({'get': 'retrieve'})),
+    url(r'^api/v2/event/(?P<slug>[-\w]+)', api_views.EventViewset.as_view({'get': 'retrieve'}, lookup_field='slug')),
```

Comment:
@GregoryHorvath This will remove this action route https://github.com/IFRCGo/go-api/blob/254ae46fc63de967255023a290be6d592cf296d2/api/drf_views.py#L279-L281

Could you move this part below this?
https://github.com/IFRCGo/go-api/blob/254ae46fc63de967255023a290be6d592cf296d2/main/urls.py#L142

Reply:
Oh okay sorry this was a merge conflict and I thought it was just an accidental move or something. Fixed in a sec.